docs/data/pages.ts

```diff
@@ -23,144 +23,171 @@ const pages: MuiPage[] = [
+              { pathname: '/x/react-data-grid/row-recipes', title: 'Recipes' },
+            ],
+          },
+          { pathname: '/x/react-data-grid/editing' },
+          { pathname: '/x/react-data-grid/sorting' },
+          {
+            pathname: '/x/react-data-grid/filtering-group',
+            title: 'Filtering',
+            children: [
+              { pathname: '/x/react-data-grid/filtering', title: 'Overview' },
+              { pathname: '/x/react-data-grid/filtering/customization' },
+              { pathname: '/x/react-data-grid/filtering/quick-filter' },
+              { pathname: '/x/react-data-grid/filtering/server-side', title: 'Server-side filter' },
+              { pathname: '/x/react-data-grid/filtering/multi-filters', plan: 'pro' },
+              {
+                pathname: '/x/react-data-grid/filtering/header-filters',
+                plan: 'pro',
+                newFeature: true,
+              },
+              { pathname: '/x/react-data-grid/filtering-recipes', title: 'Recipes' },
+            ],
+          },
+          { pathname: '/x/react-data-grid/pagination' },
+          {
+            pathname: '/x/react-data-grid/selection',
+            children: [
+              { pathname: '/x/react-data-grid/row-selection' },
+              { pathname: '/x/react-data-grid/cell-selection', plan: 'premium', newFeature: true },
+            ],
+          },
+          { pathname: '/x/react-data-grid/virtualization' },
+          { pathname: '/x/react-data-grid/accessibility' },
+          { pathname: '/x/react-data-grid/performance' },
         ],
       },
       {
-        pathname: '/x/react-data-grid/rows',
+        pathname: '/x/react-data-grid/advanced-features-group',
+        subheader: 'Advanced features',
         children: [
-          { pathname: '/x/react-data-grid/row-definition' },
-          { pathname: '/x/react-data-grid/row-updates' },
-          { pathname: '/x/react-data-grid/row-height' },
-          { pathname: '/x/react-data-grid/row-spanning', newFeature: true },
-          { pathname: '/x/react-data-grid/master-detail', plan: 'pro' },
-          { pathname: '/x/react-data-grid/row-ordering', plan: 'pro' },
-          { pathname: '/x/react-data-grid/row-pinning', plan: 'pro' },
-          { pathname: '/x/react-data-grid/row-recipes', title: 'Recipes' },
+          { pathname: '/x/react-data-grid/tree-data', plan: 'pro' },
+          { pathname: '/x/react-data-grid/row-grouping', plan: 'premium' },
+          { pathname: '/x/react-data-grid/aggregation', plan: 'premium' },
+          { pathname: '/x/react-data-grid/pivoting', plan: 'premium', planned: true },
         ],
       },
-      { pathname: '/x/react-data-grid/editing' },
-      { pathname: '/x/react-data-grid/sorting' },
       {
-        pathname: '/x/react-data-grid/filtering-group',
-        title: 'Filtering',
+        pathname: '/x/react-data-grid/visual-customization-group',
+        subheader: 'Visual customization',
```

Comment:
I wonder if this could just be **Customization**? Custom subcomponents can be more than just visual customizations. You can create totally custom experiences through slots.

Reply:
True - "Customization" felt a bit vague at first, but maybe it's good to keep more open-ended.